core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -184,12 +209,21 @@ object ConsumerPerformance {
                            lastMessagesRead: Long,
                            startMs: Long,
                            endMs: Long,
-                           dateFormat: SimpleDateFormat) = {
+                           dateFormat: SimpleDateFormat,
+                           beginTime: Option[Long] = None,
```

Comment:
`startTime` changes in each round of `printProgressMessage` but `beginTime` does not change after a test begins. `printProgressMessage` must use `startTime` to calculate TPS in a single call.

However, we also need to pass through `beginTime` which does not change after perf test gets started. It is used to calculate the total fetch time and fetch-time-related measures.

What do you think?